h/templates/client/about_this_version_dialog.html

```diff
@@ -0,0 +1,25 @@
+<div class="sheet">
+  <i class="close h-icon-close"
+    role="button"
+    title="Close"
+    ng-click="vm.aboutThisVersionDialog.visible = false"></i>
+  <div class="about-this-version">
+    <ul class="nav nav-tabs">
+      <li class="active"><a href="">About this verison</a></li>
+    </ul>
+    <dl class="tab-content">
+      <dt class="about-this-version__key">Version: </dt>
+      <dd class="about-this-version__val">{{ vm.version }}</dd>
+      <dt class="about-this-version__key">User Agent: </dt>
+      <dd class="about-this-version__val">{{ vm.userAgent }}</dd>
+      <dt class="about-this-version__key">URL: </dt>
+      <dd class="about-this-version__val">{{ vm.url }}</dd>
+      <div ng-if="vm.documentFingerprint">
+        <dt class="about-this-version__key">PDF fingerprint: </dt>
+        <dd class="about-this-version__val">{{ vm.documentFingerprint }}</dd>
+      </div>
+      <dt class="about-this-version__key">User ID: </dt>
+      <dd class="about-this-version__val">{{ vm.auth.userid.split(":")[1] }}</dd>
```

Comment:
There is a `parseAccountID` function in `scripts/filter/persona.js` which you can use for this.